src/utils/data/models.d.ts

```diff
@@ -21,3 +21,10 @@ interface Measurement {
     qualityIndex: string;
   };
 }
+
+interface AppState {
+  origin: string;
+  route: string;
+  isSSR: boolean;
+  appData: any;
+}
```

Comment:
Hm, this isn't really a model I'd say. Should it be in its own file instead maybe?